src/mushroom/Mushroom.php

```diff
@@ -57,6 +57,20 @@ public function __construct(Curl $curl = null, Canonical $canonical = null)
         $this->canonical = is_null($canonical) ? new Canonical() : $canonical;
     }
 
+    /**
+     * Add a domain to JS Redirect Domains array.
+     *
+     * @param string $domain
+     *
+     * @return $this
+     */
+    public function addJsRedirectDomain(string $domain)
```

Comment:
Unfortunately PHP5 does not support type hinting to basic data types like integers, booleans or strings. Since this package supports php 5.4 >, we'll need to get rid of that typehint. 

Also, it would be great to throw a test around this!

Reply:
Sure, I'll update the PR soon. Thanks!

Reply:
If I may ask, is there any particular reason to support older versions of PHP? I mean, they've already reached EOL. Just curious.

Reply:
Yeah I know of a few projects using it that are still PHP5. Also, there isn't much to gain for this particular package to require anything more than 5. 

Not closed to the idea, but would require a major version bump and some changes. For now I'd rather we keep for PHP5.4+